tests/test_preprints.py

```diff
@@ -336,8 +336,16 @@ def test_format_preprint(self):
 
         subjects = [nodes.pop(k) for k, v in nodes.items() if v['@type'] == 'subject']
         through_subjects = [nodes.pop(k) for k, v in nodes.items() if v['@type'] == 'throughsubjects']
-        assert sorted(subject['@id'] for subject in subjects) == sorted(tt['subject']['@id'] for tt in through_subjects)
-        assert sorted(subject['name'] for subject in subjects) == [s.bepress_text for h in self.preprint.subject_hierarchy for s in h]
+        s_ids = [s['@id'] for s in subjects]
+        ts_ids = [ts['subject']['@id'] for ts in through_subjects]
+        cs_ids = list(set(s['central_synonym']['@id'] for s in subjects))
+        for ts in ts_ids:
+            assert ts in s_ids
+            assert ts not in cs_ids  # Only aliased subjects are connected to self.preprint
+        for s in subjects:
+            subject = Subject.objects.get(text=s['name'])
+            assert s['uri'].endswith('v2/taxonomies/{}/'.format(subject._id))  # This cannot change
```

Comment:
Hard-coded here, rather than using `.absolute_api_v2_url`, to assert that it doesn't change. SHARE uses it for deduplication.